tests/utils.py

```diff
@@ -241,9 +241,22 @@ def create_jsonlines_feature_files(path):
                      features[example_num]["f{}".format(feat_num)]}
                 sub_features.append(x)
             fs = FeatureSet('ablation_cv', ids, features=sub_features, labels=labels)
+
             writer = NDJWriter(file_path, fs)
             writer.write()
 
+        # now write out the last file which is basically
+        # identical to the last featureset we wrote
+        # except that it has two extra instances
+        fs = FeatureSet('extra',
```

Comment:
I was having difficulty in understanding this. Please correct me if I am wrong
Though `create_jsonlines_feature_files` function is called from multiple locations, they are only using feature files between `f0` and `f4`. The only new test case `test_folds_file_with_fewer_ids_than_featureset` will use `f5` that has more instances than in the provided `folds_file` file.

Reply:
`create_jsonlines_feature_files()` is created from multiple locations but it only does something if any of the files is missing from disk. But you are right in that `f5.jsonlines is _only_ used by this one test and not anywhere else.